apps/avm-relayer/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "@blocksense/avm-relayer",
+  "version": "0.0.1",
+  "type": "module",
+  "license": "MIT",
+  "description": "The AVM Relayer that forwards feed data from Kafka's message queue to the ADFS contract.",
+  "bin": "./dist/bin.cjs",
+  "scripts": {
+    "build": "tsup --dts && yarn copy-package-json",
+    "ts": "yarn node --import tsx",
```

Here you can change `ts` command to `start` [the way it is merged in main](https://github.com/blocksense-network/blocksense/blob/main/apps/changelog-generator/package.json#L10)
```suggestion
   "start": "yarn node --import tsx ./src/bin.ts",
```